x-pack/plugins/infra/public/components/asset_details/tabs/overview/metrics/metrics_title.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+
+import { EuiFlexItem, EuiTitle, EuiFlexGroup } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { HostMetricsExplanationContent } from '../../../../lens';
+import { Popover } from '../../common/popover';
+
+const SectionTitle = ({ title }: { title: string }) => {
+  return (
+    <EuiTitle size="xxs">
+      <span>{title}</span>
+    </EuiTitle>
+  );
+};
```

Comment:
I wanted to avoid changing the other titles in this PR but it will be fine to just move them into a shared place and have a component for a title and for a title with a tooltip 👍